.github/workflows/lambdas-scheduled-trivy-scan.yml

```diff
@@ -0,0 +1,182 @@
+name: Run scheduled Trivy scan
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 3 * * *'
+
+jobs:
+  variables:
+    runs-on: ubuntu-latest
+
+  validate-gds:
+    strategy:
+      matrix:
+        include:
+          - module: lambdas
+            scanref: ./lambdas
+            trivyignore: lambdas/.trivyignore
+          - module: lib
+            scanref: ./lib
+            trivyignore: lib/.trivyignore
+      fail-fast: false
+    name: Validate gds
```

Comment:
This is quite picky, but I think in all the places you refer to 'gds' it might be nicer to have it as lep, as I'd say that's the equivalent to what gdx is now, rather than gds